conda_forge_webservices/update_me.py

```diff
@@ -23,14 +23,12 @@ def _run_git_command(args):
 
 
 def get_current_versions():
-    r = subprocess.run(["conda", "list"], capture_output=True)
-    out = r.stdout.decode("utf-8")
+    r = subprocess.run(["conda", "list", "--json"], capture_output=True)
```

Comment:
Should we raise any process errors (in the unlikely event they occur)?

```suggestion
    r = subprocess.run(["conda", "list", "--json"], capture_output=True, check=True)
```

Reply:
sure? The server will 500 in this case and kill the gha job.